src/TC/src/NvDecoder.cpp

```diff
@@ -200,10 +163,9 @@ int NvDecoder::HandleVideoSequence(CUVIDEOFORMAT* pVideoFormat) noexcept
     p_impl->decoder_recon++;
     CudaCtxPush ctxPush(p_impl->m_cuContext);
 
-    int nDecodeSurface =
-        GetNumDecodeSurfaces(pVideoFormat->codec, pVideoFormat->coded_width,
-                             pVideoFormat->coded_height);
-
+    // Shall be enough according to NVIDIA Nvdec mem optimization blog article;
```

Comment:
The article seems to be a useful resource justifying the `+3`. Would you mind adding the URL to the comment?

Reply:
I don't mind at all, please go ahead and add the link:
https://developer.nvidia.com/blog/optimizing-video-memory-usage-with-the-nvdecode-api-and-nvidia-video-codec-sdk/

Maybe it would be better to put links in the Wiki?

Reply:
DevTech handed this project over to R&D. @rnaskulwar could you have a look where is info would make sense in the wiki?